server/index.js

```diff
@@ -0,0 +1,72 @@
+// const express = require('express');
+import express from 'express';
+import session from 'express-session';
+import morgan from 'morgan';
+
+import bodyParser from 'body-parser';
+
+import graphqlHTTP from 'express-graphql';
+import schema from './schema';
+import db from './db';
+
+import webpack from 'webpack';
+import webpackDevMiddleware from 'webpack-dev-middleware';
+import webpackHotMiddleware from 'webpack-hot-middleware';
+
+const app = express();
+
+const PORT = 3000;
+
+const compiler = webpack(require('../webpack.config.development.js'));
+
+app.use(webpackDevMiddleware(compiler));
+app.use(webpackHotMiddleware(compiler));
+
+app.use(bodyParser.json());
+
+// Logging middleware
+app.use(morgan('dev'));
+
+app.use(session({
+    secret: 'secretkey',
+    cookie: {maxAge: 10 * 60 * 1000},
+    resave: false,
+    saveUninitalized: false,
+}));
+
+// GraphQL endpoint
+app.use('/graphql', graphqlHTTP((req) => {
+    return {
+        schema: schema,
+        rootValue: req,
+        graphiql: true
+    };
+}));
+
+// Login endpoint. This authenticates the session. We could instead do
+// this using GraphQL.
+app.post('/login', function (req, res) {
```

Comment:
I prefer using GraphQL for auth part as well, since we are going to use GraphQL. Not a fan of mixing both.